modules/apps/object/object-test/src/testIntegration/resources/com/liferay/object/web/internal/portlet/action/test/dependencies/linked_object_folder.json

```diff
@@ -0,0 +1,221 @@
+{
+	"actions": {
+		"delete": {
+		},
+		"get": {
+		},
+		"permissions": {
+		},
+		"update": {
+		}
+	},
+	"externalReferenceCode": "linked-object-folder",
+	"label": {
+		"en_US": "Linked Object Folder"
+	},
+	"name": "LinkedObjectFolder",
+	"objectFolderItems": [
+		{
+			"linkedObjectDefinition": false,
+			"objectDefinition": {
+				"accountEntryRestricted": false,
+				"accountEntryRestrictedObjectFieldName": "",
+				"actions": {
+					"delete": {
+					},
+					"get": {
+					},
+					"permissions": {
+					},
+					"publish": {
+					},
+					"update": {
+					}
+				},
+				"active": false,
+				"defaultLanguageId": "en_US",
+				"enableCategorization": true,
+				"enableComments": false,
+				"enableLocalization": false,
+				"enableObjectEntryHistory": false,
+				"externalReferenceCode": "linkedERC",
+				"label": {
+					"en_US": "linkedObject"
+				},
+				"modifiable": true,
+				"name": "LinkedObject",
+				"objectActions": [
+				],
+				"objectFields": [
+					{
+						"DBType": "String",
```

Comment:
Do we really need these system object fields? I believe they are created even though we don't pass then, right?

You can create custom object field here instead